src/udp/udp.c

```diff
@@ -200,6 +207,7 @@ static void udp_read(struct udp_sock *us, re_sock_t fd)
 	us->rh(&src, mb, us->arg);
 
  out:
+	mtx_unlock(us->lock);
```

Comment:
Is locking `us->rh` needed?

Reply:
No. If needed should be done in the upper layer with a separate mutex. So will change this.